README.md

```diff
@@ -3,17 +3,20 @@
 The official ReVanced Manager based on Flutter.
 
 ## 🔽 Download
-To download latest Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.
+To download the latest Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.
```

Comment:
This should say "To download ReVanced Manager" as "ReVanced Manager" is a product name.